tensorflow/python/keras/utils/vis_utils_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Keras Layer utils."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python import keras
+from tensorflow.python.keras.utils import vis_utils
+from tensorflow.python.lib.io import file_io
+from tensorflow.python.platform import test
+
+class ModelToDotFormatTest(test.TestCase):
+
+  def test_plot_model(self):
```

Comment:
Can you split this into two different unit tests? Otherwise LGTM to me.

Reply:
same functional test so kept under single unit test case. Hope this not required to split in to two different unit test.